core/contracts/financial_templates/FeePayer.sol

```diff
@@ -70,6 +70,22 @@ contract FeePayer is Testable {
         return regularFee.add(latePenalty);
     }
 
+    /**
+     * @notice Pays UMA DVM final fees to the Store contract. This is a flat fee charged for each price request.
+     * @return the amount of collateral that was paid to the Store.
+     */
+    function payFinalFees() public returns (FixedPoint.Unsigned memory totalPaid) {
```

Comment:
Is there any reason this should be public? Couldn't people drain the contract by repeatedly calling this method from the outside? By my understanding it should only be callable from inside the contract, right?

Reply:
ah this was just a typo in copying the code initially from `PricelessPositionManager.payFees()`

Reply:
yes good point

Reply:
this is now `internal`